CHANGELOG.md

```diff
@@ -1,5 +1,107 @@
 # @rhds/elements
 
+## 1.1.0
+
+### Minor Changes
+
+- 9eedf473: `<rh-tag>`: added `outline` variant
+- bc0fd968: ✨ Added `<rh-card>`.
+
+  Card creates a component with a header, body, and footer. The header and footer are optional.
+
+  ```html
+  <rh-card>
+    <h2 slot="header">Headline, sm</h2>
+    <p>
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam eleifend
+      elit sed est egestas, a sollicitudin mauris tincidunt. Pellentesque vel
+      dapibus risus. Nullam aliquam felis orci, eget cursus mi lacinia quis.
+      Vivamus at felis sem.
+    </p>
+    <rh-cta slot="footer">
+      <a href="#">Call to action</a>
+    </rh-cta>
+  </rh-card>
+  ```
+
+- b4815ae7: ✨ Added `<rh-audio-player>`.
+
+  Audio-player creates a custom UI for audio files.
+
+  ```html
+  <rh-audio-player>
+    <h3 slot="title">Rethinking Networks In Telecommunications</h3>
+    <p slot="series">Code Comments</p>
+    <audio crossorigin="anonymous" slot="media" controls>
+      <source type="audio/mp3" srclang="en" src="./rethinking.mp3" />
+    </audio>
+  </rh-audio-player>
+  ```
+
+- b4815ae7: ✨ Added `<rh-menu>`.
+
+  A menu listbox with keyboard controls.
+
+  ```html
+  <rh-menu>
+    <rh-button variant="link">Menuitem1</rh-button>
+    <rh-button variant="link">Menuitem2</rh-button>
+    <rh-button variant="link">Menuitem3</rh-button>
+    <rh-button variant="link">Menuitem4</rh-button>
+  </rh-menu>
+  ```
```

Comment:
Reminder to remove `rh-menu` from changelog as discussed in standup today.

```Suggestion
```

Reply:
to accomplish this, just delete that changeset file

Reply:
#1121

Reply:
Removed.